spec/logger_with_sentry_spec.rb

```diff
@@ -46,9 +46,9 @@ module Woodsman
         end
       end
 
-      expect(stdout_spy).to have_received(:info).with(/event=layer3 elapsed_time=\d\.\d+/)
-      expect(stdout_spy).to have_received(:info).with(/event=layer2 elapsed_time=\d\.\d+/)
-      expect(stdout_spy).to have_received(:info).with(/event=layer2 elapsed_time=\d\.\d+/)
+      expect(stdout_spy).to have_received(:info).with(/event=layer3 elapsed_time=\d+\.\d+/)
```

Comment:
lol, i guess we never tested this with a slower computer before. Or perhaps our previous tests enforced minimum performance as a requirement! :)